installers/sasview.spec

```diff
@@ -4,10 +4,10 @@ import sys
 from pathlib import Path
 import warnings
 import platform
-import sys
+import sysconfig
 
 block_cipher = None
-PYTHON_LOC = sys.exec_prefix
+PYTHON_PACKAGES = sysconfig.get_path('platlib')
```

Comment:
Querying a random package is a bit iffy, I guess, even if it is a core package.
It seems that `site.getsitepackages()` results are platform dependent, so probably `distutils` has the best query for `site_packages` location

```
from distutils.sysconfig import get_python_lib
PYTHON_PACKAGES  = get_python_lib()
```